dactyl-config.yml

```diff
@@ -1545,6 +1586,7 @@ pages:
         targets:
             - ja
 
+        # TODO: translate
     -   md: tutorials/use-specialized-payment-types/open-a-payment-channel-to-enable-an-inter-exchange-network.md
```

Comment:
This page is on the short list of pages I'm thinking we might remove from the English version soon, so maybe don't prioritize translating it?